sklearn/datasets/_lfw.py

```diff
@@ -151,17 +159,20 @@ def _load_imgs(file_paths, slice_, color, resize):
 
         # Checks if jpeg reading worked. Refer to issue #3594 for more
         # details.
-        img = imread(file_path)
+        pil_img = Image.open(file_path)
+        pil_img.crop((w_slice.start, h_slice.start, w_slice.stop, h_slice.stop))
+        if resize is not None:
+            pil_img = pil_img.resize((w, h))
+        img = np.asarray(pil_img)
```

Comment:
Can we directly create the 32bit `face`?
```suggestion
        face= np.asarray(pil_img, dtype=np.float32)
```

Reply:
Yup this is better. Thanks!